.github/workflows/tail-call.yml

```diff
@@ -79,23 +81,31 @@ jobs:
 
       - name: Native Windows (debug)
         if: runner.os == 'Windows' && matrix.architecture != 'ARM64'
+        shell: cmd
         run: |
-          choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }}.1.0
+          choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }}.1.5
+          set PlatformToolset=clangcl
+          set LLVMToolsVersion=19.1.5
```

Comment:
Should this use `${{ matrix.llvm }}` as well? And should the `.1.5` also be in the matrix variable rather than assumed (for every possible version of LLVM)?

Reply:
Good catch!